modules/installation-configuration-parameters.adoc

```diff
@@ -398,7 +398,7 @@ such as `io1`.
 
 |`compute.platform.aws.type`
 |The EC2 instance type for the compute machines.
-|Valid link:https://aws.amazon.com/ec2/instance-types/[AWS instance type], such as `c5.9xlarge`.
+|Valid AWS instance type, such as `m4.2xlarge`. See the following table of supported instance types.
```

Comment:
Would explicitly naming the table and/or the section that it's in be better? Not sure if this supports a11y is at is.

Same feedback for below.

Reply:
I'm making an update based on our Slack discussion and flagging that this needs to be a link when possible.